website/README.md

```diff
@@ -1,16 +1,46 @@
-# Nomad Website
+# Nomad Documentation Website
 
 [![Netlify Status](https://img.shields.io/netlify/57d01198-7abc-4c39-a89b-386a2fe7de09?style=flat-square)](https://app.netlify.com/sites/nomad-website/deploys)
 
 This subdirectory contains the entire source for the [Nomad Website](https://nomadproject.io/). This is a [NextJS](https://nextjs.org/) project, which builds a static site from these source files.
 
+<!--
+  This readme file contains several blocks of generated text, to make it easier to share common information
+  across documentation website readmes. To generate these blocks from their source, run `npm run generate:readme`
+
+  Any edits to the readme are welcome outside the clearly noted boundaries of the blocks. Alternately, a
+  block itself can be safely "forked" from the central implementation simply by removing the "BEGIN" and
+  "END" comments from around it.
+-->
```

Comment:
Woah, slick tech.

Reply:
🎉 [This is the origin](https://github.com/hashicorp/nextjs-scripts#markdown-blocks) - `npm run generate:readme` will regenerate blocks any time!